recording_server/src/streaming.ts

```
@@ -33,6 +33,8 @@ export class Streaming {
 
     // Sound level monitoring
     private currentSoundLevel: number = 0
+    private lastSoundLogTime_ms: number = 0
+    private readonly SOUND_LOG_INTERVAL_MS: number = 500 // Log maximum every 500ms (2 times per second)
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)